.gitignore

```diff
@@ -1,132 +1,6 @@
-# Logs
-logs
```

Comment:
we probably also need the previous gitignores for the frontend? please ask rutik